content/version/2/0/code_of_conduct.de.md

```diff
@@ -0,0 +1,137 @@
++++
+version = "2.0"
+aliases = ["/version/2/0/de"]
++++
+
+# Vereinbarung über Verhaltenskodex für Mitwirkende
+
+## Unsere Verpflichtung
+
+Wir als Mitglieder, Teilnehmende und Verantwortliche unserer Gemeinschaft
+verpflichten uns bei der Teilnahme an dem Projekt und unserer Gemeinschaft zu
+einer belästigungsfreien Erfahrung für alle, unabhängig von Alter, Körpergröße,
+Behinderung, ethnischer Zuordnung, geschlechtlicher Identität und Ausdruck,
+Grad der Erfahrung, Bildung, sozialem Status, Nationalität, persönlicher
+Erscheinung, Rasse, Religion oder Geschlechtsidentität und sexueller
+Orientierung.
+
+Wir verpflichten uns, in einer Weise zu handeln und zu interagieren, die zu
+einer offenen, einladenden, vielfältigen, inklusiven und gesunden Gemeinschaft
+beiträgt.
+
+## Unsere Standards
+
+Beispiele für Verhaltensweisen, die zu einem positiven Umfeld für unsere
+Gemeinschaft beitragen, sind:
+
+- Einfühlungsvermögen und Freundlichkeit gegenüber anderen Menschen zeigen
+- Unterschiedliche Meinungen, Standpunkte und Erfahrungen respektieren
+- Konstruktives Feedback geben und würdevoll annehmen
+- Verantwortung übernehmen und uns bei denjenigen entschuldigen, die von unseren
+  Fehlern betroffen sind, und aus den Erfahrungen lernen
+- Konzentration auf das, was nicht nur für uns als Individuen, sondern für die
+  gesamte Gemeinschaft das Beste ist
+
+Beispiele für inakzeptables Verhalten sind:
+
+- Die Verwendung sexualisierter Sprache, Bilder oder Symbolik sowie unerwünschte
+  Versuche sexueller Anbahnung
```

Comment:
Anbahnung sounds odd to me here. Would choose Annäherung.

Reply:
I have kept the spelling from 1.4 here, even though it sounds odd to me.
Here I would also like to ask for a short vote: 👍🏽 / 👎🏽

Reply:
After the quick vote I added your suggestion. Thank you for it.